app/layout/header.phtml

```diff
@@ -14,12 +14,9 @@ if (FreshRSS_Auth::accessNeedsAction()) {
 
 <header class="header">
 	<div class="item title">
-		<h1>
-			<a href="<?= _url('index', 'index') ?>">
-				<img class="logo" src="<?= _i('icon', true) ?>" alt="" />
-				<?= FreshRSS_Context::$system_conf->title ?>
-			</a>
-		</h1>
+		<a href="<?= _url('index', 'index') ?>">
+			<img class="logo" src="<?= _i('FreshRSS-logo', true) ?>" alt="" />
+		</a>
```

Comment:
I'm (finally) doing the update of my instance, and I realize this change breaks the possibility to change the name of the instance. I don't remember seeing someone complaining about that and I can manage this case differently, but I wonder why it was needed?

Also, if we keep this solution, the `alt` attribute should be set to "FreshRSS" to give a label to the screen readers.

Reply:
I simply overlooked it. But yes, setting aside the h1 thing I have to say I'm confused about the additional change.

Reply:
Suggestion of fix, a bit more powerful https://github.com/FreshRSS/FreshRSS/pull/4369